components/groups/windows/DisabledWindowItem.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <v-card
+    flat
+    tile
+  >
+    <v-window
+      v-model="onboarding"
+      show-arrows
+    >
+      <v-window-item
+        v-for="n in length"
+        :key="`card-${n}`"
+        disabled
+      >
+        <v-card
+          color="grey"
+          height="200"
+        >
+          <v-row
+            class="fill-height"
+            align="center"
+            justify="center"
+          >
+            <h1
+              style="font-size: 5rem;"
+              class="white--text"
+            >
+              Slide {{ n }}
+            </h1>
+          </v-row>
+        </v-card>
+      </v-window-item>
+    </v-window>
+
+    <v-card-actions class="justify-space-between">
+      <v-btn
+        text
+        @click="prev"
+      >
+        <v-icon>mdi-chevron-left</v-icon>
+      </v-btn>
+      <v-item-group
+        v-model="onboarding"
+        class="text-center"
+        mandatory
+      >
+        <v-item
+          v-for="n in length"
+          :key="`btn-${n}`"
+          v-slot="{ active, toggle }"
+        >
+          <v-btn
+            :input-value="active"
+            icon
+            @click="toggle"
+          >
+            <v-icon>mdi-record</v-icon>
+          </v-btn>
+        </v-item>
+      </v-item-group>
+      <v-btn
+        text
+        @click="next"
+      >
+        <v-icon>mdi-chevron-right</v-icon>
+      </v-btn>
+    </v-card-actions>
+  </v-card>
+</template>
+<script>
+export default {
+  data: () => ({
```

Comment:
And as we don't use custom navigation, please delete script for it

Reply:
DONE